src/tsickle.ts

```diff
@@ -126,7 +139,7 @@ class ClosureRewriter extends Rewriter {
         let newTag: jsdoc.Tag = {
           tagName: 'param',
           optional: paramNode.initializer !== undefined || paramNode.questionToken !== undefined,
-          parameterName: unescapeName(paramSym.getName()),
+          parameterName: makeLegalExternsParam(unescapeName(paramSym.getName())),
         };
```

Comment:
Ah, right. My thought would be any real change to the original values warrants at least some comment "tsickle changed x because...closure." Human readable or not...some silly human will try to read it.  Worth an issue discussion to explore, so it's consistent across the output.